config.yml

```diff
@@ -0,0 +1,13 @@
+---
+upload:
+  s3:
+    region: ap-northeast-1
+    bucket: CHANGEME
+    key_prefix: CHANGEME
+    acl: public-read
+template:
+  upload: s3
+  # see cloudstack-api listZones
+  zoneid: XXXXXXXX-XXXX-XXXX-XXXXXXXXXXXXXXXXX
+  # see http://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html
```

Comment:
this comment should be before `acl:` field of the `s3:` section as it explains what `public-read` is.